src/podrum/command/vanilla/DifficultyCommand.py

```diff
@@ -0,0 +1,39 @@
+"""
+*  ____           _
+* |  _ \ ___   __| |_ __ _   _ _ __ ___
+* | |_) / _ \ / _` | '__| | | | '_ ` _ \
+* |  __/ (_) | (_| | |  | |_| | | | | | |
+* |_|   \___/ \__,_|_|   \__,_|_| |_| |_|
+*
+* Licensed under the Apache License, Version 2.0 (the "License")
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+"""
+
+from podrum.command.Command import Command
+from podrum.command import CommandManager
+
+class HelpCommand(Command):
```

Comment:
Change name of this Class to for example
```suggestion
class DifficultyCommand(Command):
```
beacuse class HelpCommand arleady exist.

Reply:
Opsie totally forgot to do that.